### DISCUSSION TITLE
Bug Report: Ragdolling Glitches with Real-time Retargeting

### DISCUSSION THREAD

**Discussion by gustavorios2929** [Sixze/ALS-Refactored#371] [General]
I'm experiencing undesired behavior during ragdolling.
In Unreal 5, we can perform real-time retargeting using IKrig and retargeters in the "Retarget pose from mesh" node.
The issue is that with ALS refactored, when activating ragdolling, the capsule spins wildly. This doesn't affect the main mesh, but the one using real-time retargeting displays some glitchy movements, especially at the beginning of ragdolling.
Below is an image of a workaround running on tick that improves the situation by 90% (although the character still occasionally gets up spinning strangely in some scenarios).
[https://blueprintue.com/blueprint/shlzg2nc/](url)

**Comment by BITRABBBIT** [comment]
Hi. I'm facing the same problem. I would be very grateful if you would update the link to your solution, as it does not work.

**Reply by gustavorios2929** [reply]
This update resolved the issue for me:

https://github.com/Sixze/ALS-Refactored/commit/5429e5b9000455e53077d6d2d5ec69144d10769b

I have continued to apply updates since then, and everything is functioning smoothly.
